client/src/components/Polaroid.js

```diff
@@ -4,49 +4,48 @@ import defaultPic from '../assets/user-regular.svg';
 
 const Frame = styled.div`
   display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  padding: 0.2rem;
+  flex-flow: column wrap;
   background-color: #ffffff;
-  justify-content: center;
+  margin-right: 0.3rem;
   cursor: pointer;
+  width: 10.5rem;
   outline: none;
   border: none;
   box-shadow: 0rem 0.2rem 0.4rem;
 `;
 
 const PictureWrapper = styled.div`
   display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
+  flex-flow: column wrap;
+  border: none;
+  outline: none;
+  margin: 0.6rem;
+  height: 12rem;
 `;
 
 const Picture = styled.img`
   background-image: url(${defaultPic});
   background-color: ${(props) => props.theme.colors.light};
   background-repeat: no-repeat;
   background-position: center;
-  background-size: 10rem;
-  text-align: center;
-  max-width: 100%;
-  height: 300px;
-  margin: 1rem;
-  flex-grow: 1;
+  background-size: 5rem;
```

Comment:
what do you want to accomplish with this property? if you want to set the size of the image you can use width and height. -> https://developer.mozilla.org/en-US/docs/Web/CSS/background-size

Reply:
I want to resizing the background image like [here](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Resizing_background_images)